src/translib/transformer/xfmr_vrf.go

```diff
@@ -306,15 +311,11 @@ var network_instance_table_name_xfmr TableXfmrFunc = func (inParams XfmrParams)
          * For CREATE or PATCH at top level (Network_instances), check the config type if user provides one 
          * For other cases of UPATE, CREATE, or GET/DELETE, get the table name from the key only
          */ 
+        oc_nwInstType, ierr := getNwInstType(nwInstObj, keyName)
```

Comment:
just want to make sure, is ierr ==nil if the network instance type cannot be found from getNwInstType?